glimmer-version/app/components/focusing-inner.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  didUpdateAttrs() {
+    this._super(...arguments);
+    this.attrs.focus(this.get('outletState'))
```

Comment:
Can we 🔥 the `this.attrs` usage throughout?  (happy to save this for a later PR though)